src/components/result-list/exam-results.tsx

```diff
@@ -38,6 +42,7 @@ export const ExamResults = (props: ExamResultsProps) => {
             </td>
           ))}
       <td colSpan={!hasScores ? maximumQuestions + 1 : undefined}>
+        <span className="exam-technical-error">{getTechnicalError(examTechnicalErrors[0])}</span>
```

Comment:
Jos näitä on aina yksi ja käytetään aina vain ensimmäistä, niin ExamResults voisi ottaa sisään yhden vain:

```
examTechnicalError: TechnicalError
```